packages/grafana-ui/src/components/ConfirmButton/ConfirmButton.story.tsx

```diff
@@ -33,8 +33,13 @@ export default {
     closeOnConfirm: true,
   },
   argTypes: {
-    confirmVariant: { control: { type: 'select' } },
-    size: { control: { type: 'select' } },
+    confirmVariant: {
+      control: {
+        type: 'select',
+      },
+      options: ['primary', 'secondary', 'destructive', 'link'],
```

Comment:
I was under the impression Storybook would infer these from the types.

Reply:
Doesn't seem like it: https://github.com/storybookjs/storybook/issues/13380#issuecomment-739104528